docs/api/python.rst

```diff
@@ -17,7 +17,7 @@ data abstractions (blobs, folders, tables).
 
 Compute Abstractions
 ------------------------------------
-The Function, Cluster, Env, Package, Run, and Module APIs allow a seamless flow of code and execution across local and remote compute.
+The Function, Lambda Function, Cluster, Env, Package, Run, and Module APIs allow a seamless flow of code and execution across local and remote compute.
```

Comment:
In this section would be worth highlighting the two core usage paths - passing in a function callable, or the typical lambda flow where you provide paths to code and a handler function name